{{ cookiecutter.package_name }}/.coveragerc

```diff
@@ -0,0 +1,34 @@
+[tool.coverage.run]
```

Comment:
Are they the same heading in their dedicated file?

Reply:
no: https://coverage.readthedocs.io/en/latest/config.html#sample-file

```suggestion
[run]
```